doc/arkode/guide/source/Usage/LSRKStep/User_callable.rst

```diff
@@ -0,0 +1,319 @@
+
+   Returns the number of calls to the user's right-hand
+   side functions, :math:`f^E` and :math:`f^I` (so far).
+
+   **Arguments:**
+      * *arkode_mem* -- pointer to the LSRKStep memory block.
+      * *fe_evals* -- number of calls to the user's :math:`f^E(t,y)` function.
+      * *fi_evals* -- number of calls to the user's :math:`f^I(t,y)` function.
+
+   **Return value:**
+      * *ARK_SUCCESS* if successful
+      * *ARK_MEM_NULL* if the ARKStep memory was ``NULL``
+
+
+.. c:function:: int LSRKStepGetNumDomEigUpdates(void* arkode_mem, long int* ndomeigupdates);
+
+   Returns the number of Dominant Eigenvalue evaluations (so far).
+
+   **Arguments:**
+      * *arkode_mem* -- pointer to the LSRKStep memory block.
+      * *ndomeigupdates* -- number of calls to the user's ``DomEig`` function.
+
+   **Return value:**
+      * *ARK_SUCCESS* if successful
+      * *ARK_MEM_NULL* if the ARKStep memory was ``NULL``     
+
+
+.. c:function:: int LSRKStepGetMaxStageNum(void* arkode_mem, int* stagemax);
+
+   Returns the max number of stages taken (so far).
+
+   **Arguments:**
+      * *arkode_mem* -- pointer to the LSRKStep memory block.
+      * *stagemax* -- max number of stages taken.
+
+   **Return value:**
+      * *ARK_SUCCESS* if successful
+      * *ARK_MEM_NULL* if the ARKStep memory was ``NULL``  
+
+
+.. c:function:: int LSRKStepGetAverStageNum(void* arkode_mem, int* averstage);
+
+   Returns the average number of stages per step (so far).
+
+   **Arguments:**
+      * *arkode_mem* -- pointer to the LSRKStep memory block.
+      * *averstage* -- average number of stages.
+
+   **Return value:**
+      * *ARK_SUCCESS* if successful
+      * *ARK_MEM_NULL* if the ARKStep memory was ``NULL``  
+
+
+.. c:function:: int LSRKStepGetTimestepperStats(void* arkode_mem, long int* expsteps, long int* accsteps, long int* attempts, long int* fevals, long int* domeigfevals, long int* netfails, long int* stagemax, long int* ndomeigupdates, sunrealtype* sprmax, sunrealtype* sprmin);
+
+   Returns many of the most useful time-stepper statistics in a single call.
+
+   **Arguments:**
+      * *arkode_mem* -- pointer to the ARKStep memory block.
+      * *expsteps* -- number of stability-limited steps taken in the solver.
+      * *accsteps* -- number of accuracy-limited steps taken in the solver.
+      * *attempts* -- number of steps attempted by the solver.
+      * *fevals* -- number of calls to the user's :math:`f^E(t,y)` function.
+      * *domeigfevals* -- number of calls to the user's `DomEig` function.
+      * *netfails* -- number of error test failures.
+      * *stagemax* -- number of error test failures.
+      * *ndomeigupdates* -- number of Dominant Eigenvalue updates.
+      * *sprmax* -- number of error test failures.
+      * *sprmin* -- number of error test failures.
+
+   **Return value:**
+      * *ARK_SUCCESS* if successful
+      * *ARK_MEM_NULL* if the ARKStep memory was ``NULL``
```

Comment:
I think that this function should just be removed.  The current version will be immediately outdated when we add ImEx support, and it's likely that we'll want to add some other new "get" options in the near future.

I do wonder, though.  It looks like this returns *sprmax* and *sprmin*, but I don't recall seeing either of those in previous "get" routines.  If you plan to keep track of those statistics, then there should be "get" routines to retrieve them.  If you don't want to provide those "get" routines, then the internal variables to store those statistics should be deleted.
```suggestion
```